modules/apps/server/server-admin-web-test/src/testIntegration/java/com/liferay/server/admin/web/internal/portlet/action/test/EditServerMVCActionCommandTest.java

```diff
@@ -534,7 +538,32 @@ public boolean isTypeControlPanel() {
 				!cmd.equals("updateLogLevels") &&
 				!cmd.equals("updatePortalProperties")) {
 
-				try {
+				try (ConfigurationTemporarySwapper configurationTemporarySwapper =
+						 new ConfigurationTemporarySwapper(
+					"com.liferay.captcha.configuration.CaptchaConfiguration",
+					HashMapDictionaryBuilder.<String, Object>put(
+						"createAccountCaptchaEnabled", "true"
+					).put(
+						"maxChallenges", "1"
+					).put(
+						"sendPasswordCaptchaEnabled", "true"
+					).build());
+
+					CompanyConfigurationTemporarySwapper
+						companyConfigurationTemporarySwapper =
+						new CompanyConfigurationTemporarySwapper(
+							TestPropsValues.getCompanyId(),
+							CaptchaConfiguration.class.getName(),
+							new HashMapDictionaryBuilder(
+							).<String, Object>put(
+								"createAccountCaptchaEnabled", "true"
+							).put(
+								"sendPasswordCaptchaEnabled", "true"
+							).put(
+								"maxChallenges", "1"
```

Comment:
@jeffwu0724 Please sort like above.

Reply:
Alan helped to fix the order at https://github.com/liferay-core-infra/liferay-portal/pull/8752/commits/8f4dc0873e3b489503f44410b2b6fec0fa0b10db